docs/content/guides/dagster/code-references.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Linking to assets' code with code references | Dagster Docs
+---
+
+# Linking to assets' code with code references
+
+<Note>
+  This feature is considered <strong>experimental</strong> and is under active
+  development. This guide will be updated as we roll out new features.
+</Note>
+
+Attaching code reference metadata to your Dagster assets allows you to easily view those assets' source code from the Dagster UI:
+
+- **In local development**, navigate directly to the code backing an asset in your editor
+- **In Dagster+**, link to source code in your source control repository to see the full change history for an asset
```

Comment:
This isn't just for Dagster+, right? They can also do this in their OSS deployment.